phylo2vec/src/tree_vec/ops/newick/mod.rs

```diff
@@ -1,3 +1,5 @@
+use std::num::IntErrorKind;
+
 use crate::tree_vec::types::Ancestry;
 
 mod newick_patterns;
```

Comment:
The function `_get_cherries_recursive_inner` can be removed as it is no longer being used!